### ISSUE TITLE
Question: Mint.py example

### ISSUE THREAD

**Issue by writexavier** [creativequotient/automint#32]
In the mint.py example, the `remove_native_token` method is ran on `minting_receiver`.  In this example, shouldn't the native token be added to minting_receiver instead removed from it?

https://github.com/non-fungible-waveforms/automint/blob/main/examples/mint.py:93
```
for token in TOKENS:        
   token_id = f'{policy_id}.{token}'
   cust_receiver.remove_native_token(token_id, 1)       
   minting_receiver.remove_native_token(token_id, 1)
```

**Comment by creativequotient** [comment]
You're right. It should be `add_native_token` instead. Doing this would burn instead of mint tokens.
